tests/backup/backup_helper.go

```diff
@@ -392,9 +393,69 @@ func CreateBackupWithoutCheck(backupName string, clusterName string, bLocation s
 	}
 	_, err := backupDriver.CreateBackup(ctx, bkpCreateRequest)
 	if err != nil {
-		return err
+		return nil, err
```

Comment:
Here since you have added response in return, CreateBackup also return the response, so can we return this response also for CreateBackupWithoutCheck

Reply:
Good catch! I made some changes to the GetBackupUID code without realizing it, so now I need to correct both functions.

Reply:
As the response types for CreateBackup and InspectBackup are different, I am cannot return the CreateBackup response while returning the InspectBackup response.